src/goto-symex/goto_symex.h

```diff
@@ -476,4 +463,11 @@ class goto_symext
   }
 };
 
+void symex_transition(goto_symext::statet &state);
+
+void symex_transition(
+  goto_symext::statet &,
+  goto_programt::const_targett to,
+  bool is_backwards_goto);
```

Comment:
Why move them into the global namespace?

Reply:
The name and type of this function make it hard enough to collide with something else.